src/core/messages.ts

```diff
@@ -78,16 +78,16 @@ export interface QueryOptions {
 }
 
 /**
- * The parameters supplied to delete a message.
+ * The parameters supplied to a message action like delete or update.
  */
-export interface DeleteMessageParams {
+export interface ActionDetails {
```

Comment:
If we do this, are we not tying DeletionParams to ActionDetails? What about extending it, like
```
export interface DeleteMessageParams extends ActionDetails
 ```
Considering we may need to handle hard deletes later? Just leaves us a bit more flexibility, wdyt?